docs/learningLab/introToLit/acc.md

```diff
@@ -1,10 +1,22 @@
 ---
 sidebar_position: 3
 ---
-# 3. Working with Access Control Conditions 
-ACCs can be harnessed to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain condition(s) and then the network will provision signatures and decryption keys for users that meet those conditions.
+# 3. Access Control 
+![image](../../../static/img/ll_acc.png)
 
-### **Example Conditions**
+Access control control conditions can be used to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain conditions and then the network will provision signatures and decryption keys for users that meet those conditions.
+
+---
+
+## Objectives
+At completion of this reading you should be able to:
+
+- Describe what access control conditions are useful for.
+- List common types of applications that can be developed with access control.
+
+---
+
+## **Example Conditions**
 
 - User is a member of a DAO
```

Comment:
could also be helpful to link to specific examples in the docs here!

Reply:
will add a section to the bottom for addition learning resources!